crates/stdx/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "stdx"
+name = "ra_stdx"
```

Comment:
Hm, I'd rather move towards removing prefixes from other crates, they are not really useful. I think rustc auto-publishing automatically adds `rustc_ap` prefix to crates on publish. I think we should pursue the same strategy. 

More generally, rust-analyzer is not organized as a crates.io publishable library today, and that would be the sate for the foreseeable future. So, the complexity of publishing should be concentrated on the publishing side, rather than on authoring and using rust-analyzer crates.

Reply:
That would be a bit tough to implement. `rustc-ap` was easier to implement because they weren't separate crates. But we have them as seperate crates in a workspace.